tools/validators/instance_validator/tests/fake_instances/GOOD/links_increment.yaml

```diff
@@ -25,7 +25,7 @@ VIRTUAL-ENTITY-GUID:
 PHYSICAL-ENTITY-GUID:
   type: GATEWAYS/PASSTHROUGH
   code: PHYSICAL-ENTITY
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.